### PR TITLE
Implement unit and multiple axes for grafana in dashboard generator

### PR DESCRIPTION
**Pull request by jasminetealover**

### CODE REVIEW

**Review by Kontakter — APPROVED** [review]

**Review by Kontakter — APPROVED** [review]

**Comment by jasminetealover** [comment]
In last commit also added support for rightAxis

**Review by Kontakter — APPROVED** [review]

**Comment by jasminetealover** [comment]
The previous version had issues due to some local rebase conflicts. This version has been corrected and should be stable. Sorry :(

File: yt/admin/dashboard_generator/yt_dashboard_generator/backends/grafana/__init__.py [hunk 1]
```diff
@@ -356,8 +395,18 @@ def on_cell(self, cell, content):
...
 
         custom_settings = content["fieldConfig"]["defaults"]["custom"]
-        if cell.yaxis_to_label and SystemFields.LeftAxis in cell.yaxis_to_label:
-            custom_settings["axisLabel"] = cell.yaxis_to_label[SystemFields.LeftAxis]
+        if cell.yaxis_to_label:
+            if SystemFields.LeftAxis in cell.yaxis_to_label:
+                custom_settings["axisLabel"] = cell.yaxis_to_label[SystemFields.LeftAxis]
+            for override in content["fieldConfig"].get("overrides", []):
+                for prop in override.get("properties", []):
+                    if prop['id'] == 'custom.axisPlacement' and prop['value'] == 'right':
```
Comment (Kontakter) [inline yt/admin/dashboard_generator/yt_dashboard_generator/backends/grafana/__init__.py:403]: Let use double quotes here and below

File: yt/admin/dashboards/yt_dashboards/cluster_resources.py [hunk 2]
```diff
@@ -10,13 +10,29 @@
...
 try:
-    from .constants import (
-        CLUSTER_RESOURCES_DASHBOARD_DEFAULT_CLUSTER,
-        CLUSTER_RESOURCES_DASHBOARD_DEFAULT_TREE,
-    )
+    from . import constants
 except ImportError:
-    CLUSTER_RESOURCES_DASHBOARD_DEFAULT_CLUSTER = ""
-    CLUSTER_RESOURCES_DASHBOARD_DEFAULT_TREE = ""
+    constants = None
```
Comment (Kontakter) [inline yt/admin/dashboards/yt_dashboards/cluster_resources.py:15]: Add 1 empty line below, please

File: yt/admin/dashboard_generator/yt_dashboard_generator/helpers.py [hunk 3]
```diff
@@ -34,3 +34,98 @@ def pretty_dump_fixed_indent(x, offset=""):
 
 def pretty_print_fixed_indent(x):
     print(pretty_dump_fixed_indent(x))
+
```
Comment (Kontakter) [inline yt/admin/dashboard_generator/yt_dashboard_generator/helpers.py:37]: Two emply lines between functions (and you can run `ya make -t` to check linter warnings)

**Pull request closed without merging**